apps/api/src/domain/models/song/commands/create-song.command.ts

```diff
@@ -68,12 +67,18 @@ export class CreateSong implements ICommandBase {
     })
     readonly titleEnglish?: string;
 
-    @NestedDataType(ContributorAndRole, {
-        isArray: true,
-        label: 'contributions',
-        description: 'acknowledgement of all contributors who worked on this song',
-    })
-    readonly contributions: ContributorAndRole[];
+    /**
+     * TODO This property is being removed in favor of edge connections to a
+     * separate `Contributor` resource.  For now, we use a config to map in
+     * media credits. Be sure to remove this property from existing data. It can
+     * simply be ignored in sourcing V1 events.
+     */
+    // @NestedDataType(ContributorAndRole, {
```

Comment:
remove commented code

Reply:
No this is here intentionally. We need a migration to remove the property. That’s why there’s a comment on top.